persistence-cassandra/scaladsl/src/test/scala/com/lightbend/lagom/scaladsl/persistence/cassandra/TestEntityReadSide.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package com.lightbend.lagom.scaladsl.persistence.cassandra
+
+import javax.inject.Inject
+
+import akka.Done
+import akka.actor.ActorSystem
+import com.datastax.driver.core.{ BoundStatement, PreparedStatement }
+import com.lightbend.lagom.scaladsl.persistence.ReadSideProcessor.ReadSideHandler
+import com.lightbend.lagom.scaladsl.persistence.{ AggregateEventTag, EventStreamElement, ReadSideProcessor, TestEntity }
+
+import scala.collection.immutable.Seq
```

Comment:
i prefer to be more explicit about `immutable.Seq` when used in public api, because it's easy that it accidentally is lost in import refactorings/merge.

```
import scala.collection.immutable
```

```
: immutable.Seq
```